spec/classes/candlepin_spec.rb

```diff
@@ -49,6 +49,23 @@
           ])
         end
 
+        it do
+          is_expected.to contain_file('/etc/candlepin/certs/candlepin-ca.key').
+            with_group('tomcat').
+            with_mode('0640')
```

Comment:
You can test the relationship and answer @evgeni's question.
```suggestion
          is_expected.to contain_file('/etc/candlepin/certs/candlepin-ca.key').
            with_group('tomcat').
            with_mode('0640').
            that_notifies('Service[tomcat]')
```